### DISCUSSION TITLE
Pointing subdomain to an IP address

### DISCUSSION THREAD

**Discussion by ackledotdev** [is-a-dev/register#3444] [Q&A] [answered]
Hi there! I'm hosting an Express.js web server on a Raspberry Pi, and I'm wondering if I could get a subdomain and point it to my computer. Would that be possible?
If not, is there a way for me to point my github.io subdomain to my Pi?
Thanks in advance!

**Comment by andrewstech** [comment, marked as answer]
Yes this is possible. You need to use nginx as reverse proxy for express. You can then port forward your raspberry pi and use certbot to generate a free SSL certificate. 

Then on is-a-dev you need your file to be an **A** type record with your IP. You can make this file manually or via https://register.is-a.dev.

We use an express server for the notify-api service our config can be found [here](https://github.com/is-a-dev/register/blob/main/domains/notify-api.json)
